client/web/src/user/settings/auth/ExternalAccount.tsx

```diff
@@ -40,6 +40,32 @@ export const ExternalAccount: React.FunctionComponent<React.PropsWithChildren<Pr
 
     const { icon: AccountIcon } = account
 
+    let accountConnection: JSX.Element | string
+    switch (authProvider.serviceType) {
+        case 'openidconnect':
+            accountConnection = account.external ? account.external.userName : 'Not connected'
+            break
+        case 'saml':
```

Comment:
If you do not use `break` in the switch statement, it will automagically fall through, so this will be shorter, since both statements are the same:
```suggestion
        case 'openidconnect':
        case 'saml':
```